### PR TITLE
[Easy] Reward Query Final Touches

### PR DESCRIPTION
**Pull request by bh2smith**
Breaking up #202 where we noticed that the `Address` constructor does not allow fake addresses, we adjust the test data and update the query to comply more with how we expect query results to be formatted.

### CODE REVIEW

**Review by gentrexha — APPROVED** [review]
LGTM.

**Review by fleupold — APPROVED** [review]

File: queries/orderbook/batch_rewards.sql [hunk 1]
```diff
@@ -70,7 +70,7 @@ WITH observed_settlements AS (SELECT
                                    surplus + fee - reference_score as uncapped_reward_eth,
                                    -- Uncapped Reward = CLAMP_[-E, E + exec_cost](uncapped_reward_eth)
```
Comment (harisang) [inline queries/orderbook/batch_rewards.sql:71]: Is there a typo here in the comment? You write
`Uncapped Reward = CLAMP ... (uncapped_reward_eth)`

File: queries/orderbook/batch_rewards.sql [hunk 2]
```diff
@@ -81,17 +81,18 @@ WITH observed_settlements AS (SELECT
                               FROM participation_data
                               GROUP BY participant),
      primary_rewards as (SELECT rpt.solver,
-                                SUM(capped_reward)  as total_reward_eth,
-                                SUM(execution_cost) as total_exececution_cost_eth
+                                SUM(capped_payment) as payment_wei,
+                                SUM(execution_cost) as exececution_cost_wei
```
Comment (harisang) [inline queries/orderbook/batch_rewards.sql:85]: Is the typo here deliberate?

File: queries/orderbook/batch_rewards.sql (hunk 1 above)
Comment (bh2smith) [reply queries/orderbook/batch_rewards.sql:71]: Ya definitely. Left side should be capped reward. Gonna have to fix this later.

File: queries/orderbook/batch_rewards.sql (hunk 2 above)
Comment (bh2smith) [reply queries/orderbook/batch_rewards.sql:85]: Of course not. Just a regular old unintentional typo.

**Pull request merged**